src/wallet/wallet.cpp

```diff
@@ -2807,11 +2807,11 @@ bool CWallet::CreateTransactionInternal(
             CFeeRate discard_rate = GetDiscardRate(*this);
 
             // Get the fee rate to use effective values in coin selection
-            CFeeRate nFeeRateNeeded = GetMinimumFeeRate(*this, coin_control, &feeCalc);
+            coin_selection_params.effective_fee = GetMinimumFeeRate(*this, coin_control, &feeCalc);
```

Comment:
It seems to me that `effective_fee` is a feerate, not a fee.

Reply:
Renamed to `m_effective_feerate`.